packages/playwright-ct-react/registerSource.mjs

```diff
@@ -55,11 +69,5 @@ function render(component) {
 }
 
 window.playwrightMount = component => {
-  if (!document.getElementById('root')) {
-    const rootElement = document.createElement('div');
-    rootElement.id = 'root';
-    document.body.append(rootElement);
-  }
   ReactDOM.render(render(component), document.getElementById('root'));
```

Comment:
btw. `ReactDOM.render` is btw. deprecated in React 18, not sure which React version gets picked, but I think it will yield to console warnings if used with React 18.

Reply:
Thanks! I'll follow up!